cli/providers/aws/translate.go

```diff
@@ -354,11 +355,15 @@ func loadIAM(t *gocf.Template, s *service.Service, e *environment.Environment) {
 }
 
 func loadLogGroup(t *gocf.Template, s *service.Service, e *environment.Environment) {
-	logGroup := &gocf.LogsLogGroup{
-		LogGroupName:    gocf.String(fmt.Sprintf("i2kit-%s", s.GetFullName(e, "-"))),
-		RetentionInDays: gocf.Integer(30),
+	randomID := uuid.New()
+	logGroupResource := &gocf.Resource{
+		Properties: &gocf.LogsLogGroup{
+			LogGroupName:    gocf.String(fmt.Sprintf("i2kit-%s-%d", s.GetFullName(e, "-"), randomID.ID())),
```

Comment:
do we risk running into 'name too long' issues? Also, maybe use a timestamp instead of a random ID so a user can sort one from the other by names

Reply:
I am planning to use the service ID, but it is not there yet.
Log group names can be 512 chars length, I wouldn't worry about that.
I will create an issue to revisit every name we create and to switch to service ID once it is integrated.